Formula/fheroes2.rb

```diff
@@ -0,0 +1,39 @@
+class Fheroes2 < Formula
+  desc "Free Heroes of Might and Magic II () is a recreation of HoMM2 game engine"
+  homepage "https://ihhub.github.io/fheroes2/"
+  url "https://github.com/ihhub/fheroes2/archive/refs/tags/0.9.7.tar.gz"
+  sha256 "02e777485906b8112636173c7e241d2824b2dce7707b5ebf6b3bab70ea9865f7"
+  license "GPL-2.0-or-later"
+  head "https://github.com/ihhub/fheroes2.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :git
```

Comment:
```suggestion
```

`brew livecheck` already uses the `Git` strategy for this URL by default, so we don't need `strategy` here.